src/sentry/static/sentry/app/views/settings/components/tag.tsx

```diff
@@ -1,22 +1,27 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
 import InlineSvg from 'app/components/inlineSvg';
+import {Theme} from 'app/utils/theme';
 
-const getMarginLeft = p => {
-  if (!p.inline) {
-    return '';
-  }
-  return `margin-left: ${p.size === 'small' ? '0.25em' : '0.5em'};`;
+type Props = React.HTMLAttributes<HTMLDivElement> & {
+  priority?: keyof Theme['badge'] | keyof Theme['alert'];
```

Comment:
I would add an export in the theme like : 

```
export type Bagde = keyof typeof badges;
export type Alert = keyof typeof alerts;
```

Reply:
👎 would prefer not to pollute our namespace with additional types. Especially since we already have a component called alert.

What's the problem with doing this?